TwitterImagePipelineTests/TIPXWebPCodec.m

```diff
@@ -438,3 +427,75 @@ - (TIPImageContainer *)_tipx_renderImage
     }
     return [[TIPImageContainer alloc] initWithImage:image];
 }
+
+static BOOL TIPXWebPPictureImport(WebPPicture *picture, CGImageRef imageRef)
+{
+    __block vImage_Buffer convertedImageBuffer = {};
+
+    tipx_defer(^{
+        if (convertedImageBuffer.data) {
+            free(convertedImageBuffer.data);
+        }
+    });
+
+    if (!TIPXWebPCreateRGBADataForImage(imageRef, &convertedImageBuffer)) {
+        return NO;
+    }
+
+    return WebPPictureImportRGBA(picture, convertedImageBuffer.data, (int)convertedImageBuffer.rowBytes);
+}
+
+static BOOL TIPXWebPCreateRGBADataForImage(CGImageRef sourceImage, vImage_Buffer *convertedImageBuffer)
+{
+    if (convertedImageBuffer == NULL) {
+        return NO;
+    }
+
+    vImage_CGImageFormat sourceImageFormat = {
+        .bitsPerComponent = (uint32_t)CGImageGetBitsPerComponent(sourceImage),
+        .bitsPerPixel = (uint32_t)CGImageGetBitsPerPixel(sourceImage),
+        .colorSpace = CGImageGetColorSpace(sourceImage),
+        .bitmapInfo = CGImageGetBitmapInfo(sourceImage)
```

Comment:
This is returning CGImageAlphaInfo value (same as CGImageGetAlphaInfo returns) --- likely a bug in Apple's framework failing to properly convert the CGImageAlphaInfo to CGBitmapInfo.  I think you should use CGImageGetAlphaInfo and do the conversion yourself - paying attention to if you want to keep alpha or not.